# Logitech F710 XInput mode
# to check your joy control: sudo jstest /dev/input/js0

axis_linear: 1  # Left thumb stick vertical
scale_linear: 0.7
axis_angular: 0  # Left thumb stick horizontal
scale_angular: 0.4

axis_vertical: 4  # Right thumb stick vertical
scale_vertical: 1.5
axis_horizontal: 3  # Right thumb stick horizontal
scale_horizontal: 1.0

enable_trigger: 2  # Left trigger (LT) button
enable_turbo_trigger: 5  # Right trigger (RT) button

button_map:
  A: 0
  B: 1
  X: 2
  Y: 3
  L: 4
  R: 5
